editor/editor_help.cpp

```diff
@@ -1831,6 +1831,15 @@ void EditorHelp::_update_doc() {
 			class_desc->push_indent(1);
 			if (!cd.properties[i].description.strip_edges().is_empty()) {
 				_add_text(DTR(cd.properties[i].description));
+				// Add copy note to built-in properties returning Packed*Array.
+				if (!cd.is_script_doc &&
+						(cd.properties[i].type == "PackedByteArray" || cd.properties[i].type == "PackedColorArray" || cd.properties[i].type == "PackedFloat32Array" ||
+								cd.properties[i].type == "PackedFloat64Array" || cd.properties[i].type == "PackedInt32Array" || cd.properties[i].type == "PackedInt64Array" ||
+								cd.properties[i].type == "PackedStringArray" || cd.properties[i].type == "PackedVector2Array" || cd.properties[i].type == "PackedVector3Array")) {
```

Comment:
This is the part where an opposite of Variant's `get_type_name` would come extremely useful, let's take notes.

Reply:
Indeed, will probably use a vector of map here like already done with types that differ in C#, but did a simple solution for now, and will see how I finally apply it

Reply:
There seems to be some interest in https://github.com/godotengine/godot/pull/85474 so that may be worth keeping in mind here.